platforms/macos/Cargo.toml

```
@@ -13,6 +13,13 @@ edition = "2021"
 [dependencies]
 accesskit = { version = "0.8.1", path = "../../common" }
 accesskit_consumer = { version = "0.11.0", path = "../../consumer" }
-objc2 = "0.3.0-beta.3"
+objc2 = "0.3.0-beta.4"
```

I'm on holiday at the moment, will gladly review it later.

To fix it in the interim, you should use `objc2 = "=0.3.0-beta.3"` instead, at least until I've released a stable version.

---

Though even with this, you should use "`=`" so that it doesn't break again in the future (this is just a quirk in how pre-release versions work)

```suggestion
objc2 = "=0.3.0-beta.4"
```